improver_tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -693,20 +691,23 @@ def setUp(self):
         self.cubes, self.cubes_no_lightning = set_up_wxcubes()
         self.wxcode = np.array(list(WX_DICT.keys()))
         self.wxmeaning = " ".join(WX_DICT.values())
+        self.no_lightning_wxcode = np.array(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Too many positional arguments for method call](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=4901764)